teapots/common/ndk_helper/CMakeLists.txt

```diff
@@ -1,16 +1,40 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11 -Wall -fno-exceptions -fno-rtti")
```

Comment:
Looks like you dropped this.

Reply:
really, let me check, it is strange to drop this

Reply:
it is there, just moved it to top of the file, right after project() statement.